include/hyphadao.hpp

```diff
@@ -23,6 +27,9 @@ namespace hyphaspace
       // v2 data structure will use variants for more generic support
       typedef std::variant<name, string, asset, time_point, uint64_t> flexvalue1;
 
+      //using flexvalue = document_graph::flexvalue; causes seg fault while compiling
+      using flexvalue = std::variant<name, string, asset, time_point, int64_t, checksum256>;
```

Comment:
We can use the typedef Content::FlexValue instead of redefining it here.  (We need to merge some changes from the newer DocumentGraph branch for this.)

Reply:
Yeah, that was my first attempt as you can see. But after getting some no sense compiling errors I just decided to go with a duplicated typedef. I'll test with the new API once it's on and see if it works.